e2e/libs/workload/deployment.py

```diff
@@ -9,7 +9,7 @@
 from utility.utility import get_retry_count_and_interval
 
 
-def create_deployment(name, claim_name):
+def create_deployment(name, claim_name, replicaset=1):
```

Comment:
Is it possible to make the function argument name identical to the field name? So we can have:
```
manifest_dict['spec']['replicas'] = replicas
```
I think it could make users lives easier.

Reply:
I prefer to use `replicaset` in this context because it aligns with the actual behavior which corresponds with the CRD kind ([ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/)) that is used by the controller. Additionally, it also simplifies when searching in the code base considering that Longhorn also uses the term `replica` for volume replicas. So this makes the distinction between the two.